src/invertObj.js

```diff
@@ -5,6 +5,9 @@ var keys = require('./keys');
 /**
  * Returns a new object with the keys of the given object
  * as values, and the values of the given object as keys.
+ * Note that the last key found is prefered when handling
+ * the same value. Non-primitive values are converted to
+ * strings which might yield undesired results.
```

Comment:
Why "non-primitive"? Primitives such as `true` and `42` are also coerced to strings. Perhaps this sentence should read:

_Values of the input object are coerced to strings to determine the keys of the output object._